src/components/CreateCoursePage/index.jsx

```diff
@@ -135,6 +138,7 @@ CreateCoursePage.propTypes = {
   }),
   fetchOrganizations: PropTypes.func,
   createCourse: PropTypes.func,
+  formValues: PropTypes.shape({}),
```

Comment:
Do we care to structure this more than just any object?

Reply:
I didn't end up bothering because we use the same situation on the EditCoursePage/Form and to list all the fields there felt like a lot. It's doable here if you think there's value in it because it is only 5 or so fields, but I figured just declaring it as an object would be good